dxr/plugins/clang/dxr-index.cpp

```diff
@@ -600,21 +600,15 @@ class IndexConsumer : public ASTConsumer,
                methodDecl->begin_overridden_methods(),
                end = methodDecl->end_overridden_methods();
              iter != end; ++iter) {
-          const FunctionDecl *overriddenDecl = nullptr;
-          // Get the overridden definition if it exists...
-          (*iter)->isDefined(overriddenDecl);
```

Comment:
Because we got rid of isDefined (which scribbles on its arg, IIRC), are we going to start finding declarations rather than definitions?